src/mds/MDSDaemon.cc

```diff
@@ -1354,9 +1354,10 @@ bool MDSDaemon::ms_verify_authorizer(Connection *con, int peer_type,
       std::ostringstream errstr;
       if (!s->auth_caps.parse(g_ceph_context, auth_cap_str, &errstr)) {
         dout(1) << __func__ << ": auth cap parse error: " << errstr.str()
-		<< " parsing '" << auth_cap_str << "'" << dendl;
-	clog->warn() << name << " mds cap '" << auth_cap_str
-		     << "' does not parse: " << errstr.str();
+             << " parsing '" << auth_cap_str << "'" << dendl;
+        clog->warn() << name << " mds cap '" << auth_cap_str
+             << "' does not parse: " << errstr.str();
+        is_valid = false;
```

Comment:
Please revert the unwanted alignment changes for others to identify your actual changes. If you make a code change on that line, then changing alignment is fine. Otherwise, don't do it. Thanks.